python/src/LeastSquaresEquationsSolver_doc.i.in

```diff
@@ -0,0 +1,47 @@
+%feature("docstring") OT::LeastSquaresEquationsSolver
+"Least Square Solver (LeastSquaresEquationsSolver) for a system of non-linear equations.
+
+
+Parameters
+----------
+absError : positive float
+    Absolute error: distance between two successive iterates at the end point.
```

Comment:
The significant fact is that this is an absolute tolerance on x, not f(x) isn'it?
```suggestion
    Absolute error: distance between two successive x iterates at the end point.
```